source/nodejs/adaptivecards-designer/webpack.config.js

```diff
@@ -87,8 +87,8 @@ module.exports = (env, argv) => {
                     {
                         dest: 'dist/adaptivecards-designer.css',
                         src: [
-                            './node_modules/adaptivecards-controls/dist/adaptivecards-controls.css',
-                            './node_modules/adaptivecards/dist/adaptivecards-carousel.css',
+                            '../node_modules/adaptivecards-controls/dist/adaptivecards-controls.css',
```

Comment:
how did it work before? it was broken?

Reply:
as part of the move to `workspaces`, the `adaptivecards-controls` package started getting linked in the project root directory :)